webapp/src/js/enketo/widgets/phone-widget.js

```diff
@@ -45,13 +56,13 @@ FormModel.prototype.types.tel = {
    * @extends Widget
    */
 class PhoneWidget extends Widget {
-  constructor( element, options, Settings = window.CHTCore.Settings ) {
-    super(element, options);
-
+  _init() {
```

Comment:
The Enketo Widget class has [an _init function](https://github.com/enketo/enketo/blob/351944c491cd59c6a0574d40393a4bb9fca7f81e/packages/enketo-core/src/js/widget.js#L48) that is intended to be overridden to include initialization logic.  We use that method in our other custom widgets, so just cleaning up this old code to follow the new convention.